### PR TITLE
add video headers and post

### PR DESCRIPTION
**Pull request by hougasian**

### CODE REVIEW

File: _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    width: auto;
+    height: auto;
+    z-index: -100;
+    transform: translate(-50%, -50%);
+  }
+```
+
+## Summary
+
+Enjoy adding video to your jekyll project. Video content in this post is from the good folks at [DISTILL](http://www.wedistill.io/).
```
Comment (kendaleiv) [inline _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md]: Could you capitlize `jekyll`?

Comment (kendaleiv) [inline _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md]: I think `DISTILL` should be `Distill`.

File: _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	poster="{{ postVideo }}.png">
+<source src="{{ postVideo }}.mp4" type="video/mp4">
+<source src="{{ postVideo }}.webm" type="video/webm">
+</video>
+```
+{% endraw %}
+
+### What options do I have?
+
+The [HTML5 Video Events and API](https://www.w3.org/2010/05/video/mediaevents.html) page is a good place to start. You can see everything the API exposes and what may suite your needs. Check out this link for more on the [element itself and attribute definitions](https://www.w3.org/TR/html5/embedded-content-0.html#the-video-element).
```
Comment (kendaleiv) [inline _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md]: `suite` should be `suit`.

File: _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+Let's load the video in our front matter, and handle fallback and poster image in the post in our include. We're only dealing with `.mp4` and `.webm` here. Video conversion? [Cloud convert](https://cloudconvert.com/mp4-to-webm) does a decent job for now. So now we have:
+
+```
+---
+layout: post
+video: /video/clouds_over_the_mountain
+---
+```
+We could have used a collection and repeated our 3 formats with the same name, but a single entry with now extention makes more sense while listing the path.
```
Comment (kendaleiv) [inline _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md]: `extention` should be `extension`. Could you clarity what "with now extension" means?

File: _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+title: "Adding video backgrounds with jekyll front-matter"
+date: 2016-11-10 13:27:57
+tags:
+- jekyll
+- video
+- liquid
+authors: Kevin Hougasian
+video: /video/liquid
+---
+Most of our web properties run on [jekyll](https://jekyllrb.com), [ebrokersoftware.com](https://ebrokersoftware.com), [theagentsurvivalguide.com](https://theagentsurvivalguide.com), and more slated for the coming year.
```
Comment (kendaleiv) [inline _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md]: Could you capitalize `jekyll`?

File: _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Adding video backgrounds with jekyll front-matter"
```
Comment (kendaleiv) [inline _posts/2016-11-10-adding-video-backgrounds-with-jekyll-front-matter.md]: Can we capitalize `jekyll`?

**Comment by kendaleiv** [comment]
We may want to disable this for mobile users, since they may be on a slow network and/or non-unlimited data plan. Thoughts? @hougasian @khalidabuhakmeh

**Comment by khalidabuhakmeh** [comment]
On iOS the video only plays when you start it.

**Comment by kendaleiv** [comment]
The video plays automatically on Chrome on my Android phone.

**Pull request merged**